### ISSUE TITLE
getting started

### ISSUE THREAD

**Issue by nousacademy** [CodeYellowBV/run-headless-chromium#8]
Im not to sure how to launch chrome with this. I get the error:
```
throw getNotFoundError(cmd)
        ^
Error: not found: chromium-browser
```
How would I use this with Chrome??

**Comment by albertlinde** [comment]
Ensure you have chromium, ex:
sudo apt-get install chromium-browser

**Comment by nousacademy** [comment]
I installed with brew: `brew cask install chromium` and also tried `brew cask install google-chrome`

**Comment by Rob--W** [comment]
I could fix this in the package by adding `/Applications/Google Chrome.app/Contents/MacOS/Google Chrome` and `/Applications/Chromium.app/Contents/MacOS/Chromium` to the list of programs, but that would be pointless if Xvfb is not available.

Can you open your terminal and see if `Xvfb -help` prints some reasonable output? This would then indicate that Xvfb is installed and that adding the path to Chrome on Mac OS would make sense.

**Comment by nousacademy** [comment]
Prints out: `command not found: Xvfb`. I just searched on SO and this result came up: http://stackoverflow.com/questions/25451133/xvfb-run-on-os-x. I always have issues with virtual frame buffer... blah

**Comment by Rob--W** [comment]
Let me know when you get Xvfb to work on Mac, then I'll publish an update that recognizes Chrome on Mac.

**Comment by nousacademy** [comment]
I got it working on mac I have xvfb and chromium

**Comment by Rob--W** [comment]
@nousacademy Is Chrome located in one of the directories from https://github.com/CodeYellowBV/run-headless-chromium/issues/8#issuecomment-224648536 ?

**Comment by nousacademy** [comment]
I get this with that command :

> use: X [:<display>] [option]
> -a #                   default pointer acceleration (factor)
> -ac                    disable access control restrictions
> -audit int             set audit trail level
> -auth file             select authorization file
> -br                    create root window with black background
> +bs                    enable any backing store support
> -bs                    disable any backing store support
> -c                     turns off key-click
> c #                    key-click volume (0-100)
> -cc int                default color visual class
> -nocursor              disable the cursor
> -core                  generate core dump on fatal error
> -dpi int               screen resolution in dots per inch
> -dpms                  disables VESA DPMS monitor control
> -deferglyphs [none|all|16] defer loading of [no|all|16-bit] glyphs
> -f #                   bell base (0-100)
> -fc string             cursor font
> -fn string             default font name
> -fp string             default font path
> -help                  prints message with these options
> -I                     ignore all remaining arguments
> -ld int                limit data space to N Kb
> -lf int                limit number of open files to N
> -ls int                limit stack space to N Kb
> -nolock                disable the locking mechanism
> -nolisten string       don't listen on protocol
> -noreset               don't reset after last client exists
> -background [none]     create root window with no background
> -reset                 reset after last client exists
> -p #                   screen-saver pattern duration (minutes)
> -pn                    accept failure to listen on all ports
> -nopn                  reject failure to listen on all ports
> -r                     turns off auto-repeat
> r                      turns on auto-repeat
> -render [default|mono|gray|color] set render color alloc policy
> -retro                 start with classic stipple and cursor
> -s #                   screen-saver timeout (minutes)
> -seat string           seat to run on
> -t #                   default pointer threshold (pixels/t)
> -terminate             terminate at server reset
> -to #                  connection time out
> -tst                   disable testing extensions
> ttyxx                  server started from init on /dev/ttyxx
> v                      video blanking for screen-saver
> -v                     screen-saver without video blanking
> -wm                    WhenMapped default backing-store
> -wr                    create root window with white background
> -maxbigreqsize         set maximal bigrequest size
> +xinerama              Enable XINERAMA extension
> -xinerama              Disable XINERAMA extension
> -dumbSched             Disable smart scheduling, enable old behavior
> -schedInterval int     Set scheduler interval in msec
> -sigstop               Enable SIGSTOP based startup
> +extension name        Enable extension
> -extension name        Disable extension
> -query host-name       contact named host for XDMCP
> -broadcast             broadcast for XDMCP
> -multicast [addr [hops]] IPv6 multicast for XDMCP
> -indirect host-name    contact named host for indirect XDMCP
> -port port-num         UDP port number to send messages to
> -from local-address    specify the local address to connect from
> -once                  Terminate server after one session
> -class display-class   specify display class to send in manage
> -cookie xdm-auth-bits  specify the magic cookie for XDMCP
> -displayID display-id  manufacturer display ID for request
> [+-]accessx [ timeout [ timeout_mask [ feedback [ options_mask] ] ] ]
>                        enable/disable accessx key sequences
> -ardelay               set XKB autorepeat delay
> -arinterval            set XKB autorepeat interval
> -screen scrn WxHxD     set screen's width, height, depth
> -pixdepths list-of-int support given pixmap depths
> +/-render		   turn on/off RENDER extension support(default on)
> -linebias n            adjust thin line pixelization
> -blackpixel n          pixel value for black
> -whitepixel n          pixel value for white
> -fbdir directory       put framebuffers in mmap'ed files in directory
> -shmem                 put framebuffers in shared memory

**Comment by Rob--W** [comment]
I just published 0.0.14, give it a try!
